noether_simulator/cfg/NoetherSimulator.cfg

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+PACKAGE = "noether_simulator"
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+import math
+
+gen = ParameterGenerator()
+
+to = gen #.add_group("toggle_options")
+
+to.add("display_sigma", double_t, 0, "display_sigma", 1.0, 0.1, 10.0)
+to.add("tool_radius", double_t, 0, "tool_radius", 0.05, 0.001, 1.0)
+to.add("process_rate", double_t, 0, "process base deposition/removal rate", 5.0, 0.001, 20.0)
+to.add("tool_height", double_t, 0, "tool_height", 0.1, 0.001, 1.0)
+
+exit(gen.generate(PACKAGE, "noether_simulator", "NoetherSimulator"))
```

Comment:
@drchrislewis Is this being used? We can probably address this later.

Reply:
Leave for later